README.md

```diff
@@ -207,7 +260,7 @@ import AuthenticateOptions from 'web-eid/models/AuthenticateOptions';
 ### ES module for browsers
 
 1. Configure the web server to expose `node_modules/web-eid/dist/es/web-eid.js`
-2. Import the modules and specify the file extension
+2. Import the modules specifying the file extension:
```

Comment:
I liked the previous wording more. Now it sounds like "if you wish to specify the file extension, import the modules".  
But maybe this isn't even something worth mentioning. The example shows how to import the module.
```suggestion
2. Import the module:
```

Reply:
Agreed, done.